README.md

```diff
@@ -28,12 +28,14 @@ script/bootstrap
 
 ### With the current API
 
-To start Docker, run all backing services including a local copy of the Accredited Programmes API (port 9091) and the Prison API (port 9092), and then the application itself, run:
+To start Docker, run all backing services including a local copy of the Accredited Programmes API (port 9091), the Prison API (port 9092), a mocked version of the Prisoner Offender Search (see below), and then the application itself, run:
```

Comment:
I think the `and` should still be there since `a local copy` applies to both the AcP and Prison (Register) APIs (and not the rest of the list)

Also, it should be `Prison Register API` or `Prison Register`, but maybe that's a separate fix